tests/layer_tests/mo_python_api_tests/test_mo_convert_extensions.py

```diff
@@ -120,3 +126,99 @@ def test_mo_convert_extensions(self, params, ie_device, precision, ir_version,
         test_params = params['params_test']
         test_params.update({'input_model': onnx_net_path})
         self._test_by_ref_graph(temp_dir, test_params, params['ref_graph'])
+
+
+def create_tf_model():
+    import tensorflow as tf
+
+    tf.compat.v1.reset_default_graph()
+
+    with tf.compat.v1.Session() as sess:
+        inp1 = tf.compat.v1.placeholder(tf.float32, [1, 2, 3], 'Input')
+        inp2 = tf.compat.v1.placeholder(tf.float32, [1, 2, 3], 'Input')
+        relu = tf.nn.relu(inp1 + inp2, name='Relu')
+
+        output = tf.nn.sigmoid(relu, name='Sigmoid')
+
+        tf.compat.v1.global_variables_initializer()
+        tf_net = sess.graph_def
+    return tf_net
+
+
+def create_ref_model_1():
+    shape = [1, 2, 3]
+    shape = PartialShape(shape)
+    param1 = ov.opset10.parameter(shape)
+    param2 = ov.opset10.parameter(shape)
+    add = ov.opset10.add(param1, param2)
+    relu = ov.opset10.relu(add)
+    sin = ov.opset10.sin(relu)
+    sigm = ov.opset10.sigmoid(sin)
+    parameter_list = [param1, param2]
+    ref_model = Model([sigm], parameter_list, "test")
+    return ref_model
+
+
+def create_ref_model_2():
+    shape = [1, 2, 3]
+    shape = PartialShape(shape)
+    param1 = ov.opset10.parameter(shape)
+    param2 = ov.opset10.parameter(shape)
+    add = ov.opset10.add(param1, param2)
+    relu = ov.opset10.relu(add)
+    sin = ov.opset10.sin(relu)
+    sigm = ov.opset10.sigmoid(sin)
+    tanh = ov.opset10.tanh(sigm)
+    parameter_list = [param1, param2]
+    ref_model = Model([tanh], parameter_list, "test")
+    return ref_model
+
+
+class LegacyExtTest(unittest.TestCase):
+    test_directory = os.path.dirname(os.path.realpath(__file__))
+    def test_legacy_extensions(self):
+        from openvino.tools.mo import convert_model
+        with tempfile.TemporaryDirectory(dir=self.test_directory) as tmpdir:
+            ext_path1 = os.path.join(os.path.dirname(__file__), "test_legacy_exts/test_exts_dir1")
+            ext_path2 = os.path.join(os.path.dirname(__file__), "test_legacy_exts/test_exts_dir2")
+            model = create_tf_model()
+            out_xml = os.path.join(tmpdir, "model.xml")
+
+            # tests for convert_model()
+            ov_model = convert_model(model, extensions=ext_path1)
+            flag, msg = compare_functions(ov_model, create_ref_model_1(), False)
+            assert flag, msg
+
+            ov_model = convert_model(model, extensions=[ext_path1, ext_path2])
+            flag, msg = compare_functions(ov_model, create_ref_model_2(), False)
+            assert flag, msg
+
+            ov_model = convert_model(model, extensions=','.join([ext_path1, ext_path2]))
+            flag, msg = compare_functions(ov_model, create_ref_model_2(), False)
```

Comment:
may be it is better to move it into `test_mo_convert_legacy_extensions.py`?

Reply:
Done.